pkg/kubelet/apis/config/types.go

```diff
@@ -379,10 +379,10 @@ type KubeletConfiguration struct {
 	// EnableSystemLogHandler enables /logs handler.
 	EnableSystemLogHandler bool
 	// ShutdownGracePeriod specifies the total duration that the node should delay the shutdown and total grace period for pod termination during a node shutdown.
-	// Defaults to 30 seconds, requires GracefulNodeShutdown feature gate to be enabled.
+	// Defaults to 0 seconds.
```

Comment:
We're adding a new `+featureGate=GracefulNodeShutdown` tag - if you put it here, later tooling will use it automatically.

Reply:
thanks. I can add that.

Should the `featureGate=GracefulNodeShutdown` tag be added here or in https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/config/v1beta1/types.go (where there's I see there's already existing tags like `+optional`)?